pytest.ini

```diff
@@ -13,6 +13,8 @@ env =
     FRESH_DESK_PRODUCT_ID=42
     FRESH_DESK_API_URL=https://example.com/freshdesk
     FRESH_DESK_API_KEY=freshdesk-api-key
+    ##### Define URIs before running local tests #####
     # SQLALCHEMY_DATABASE_URI=postgresql://postgres:LocalPassword@db:5432/test_notification_api
+    # SQLALCHEMY_DATABASE_READER_URI=postgresql://postgres:LocalPassword@db:5432/test_notification_api
```

Comment:
Could we set `SQLALCHEMY_DATABASE_URI=postgresql://postgres@localhost:5432/test_notification_api`

That should be a sensible default for developers and should work out of the box. Otherwise, what's the recommended way? Do you manually change your `.env` before running tests?

Can we not set `SQLALCHEMY_DATABASE_READER_URI` and tests will run or does it need to have a user? What about setting it to the same value (defeats the purpose or testing a read-only database user I know - but this is checked on GitHub Actions)

Reply:
I don't mind if we can have reader URI default to run as writer when not specified.

It needs to have a valid user, the initial connection is reused to create the test databases. I'm open to either @AntoineAugusti. 

I felt it needs to be explicitly changed by the developer, since we have the python-dotenv package installed which always looks for the .env, and often a developer may have a working dev configuration, which is not a good test configuration due to the database specificity. The .env will overwride the pytest, and either the developer needs to clear or change the .env to run clean tests. If we have a sensible default, it would work, but it may also lead to ambiguity because it's specified, but not looking there for the most common developer config using .env

Ironically, it would be nice to have the python-dotenv engage via its own env flag.